sectionctl.go

```diff
@@ -102,8 +114,9 @@ func main() {
 		kong.UsageOnError(),
 		kong.ConfigureHelp(kong.HelpOptions{Tree: true}),
 	)
-	bootstrap(cli, ctx)
-	err := ctx.Run()
+	contxt := bootstrap(cli, ctx)
+	ctx.BindTo(contxt, (*context.Context)(nil))
```

Comment:
Why would you ever want to pass a nil context? I don't understand why this is needed.

Reply:
I'm not sure exactly what `BindTo` does, but it does not work without it. This was how code on the internet. My best guess it that it creates a reference to the context object so that Kong passes the context by reference. This would make mutations affect the base object.